### PR TITLE
UG patron format changes

### PR DESCRIPTION
**Pull request by aditi2313**
Patron commands' format to match book commands' format

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
+**Example:**
+
+Before entering the command, all patrons are displayed.
+
+![patron-list-1](images/patron-list-1.png)
+
+After entering the command, only patrons with overdue books are displayed.
+
+![patron-list-2](images/patron-list-2.png)
+
```
Comment (yl-ang) [inline docs/UserGuide.md:330]: This line is logical in explaining to the readers of what to expect from running the patron overdue command :100:

File: docs/UserGuide.md [hunk 2]
```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
+#### 3.2.4. Finding patrons: `patron find`
+
+To find patrons in LibTask based on patron name, you can enter the `find` command with the format shown below.
+
+Format: `patron find KEYWORD [KEYWORD]…​`
+
+**Noted about the find command:**
```
Comment (yuqitanyq) [inline docs/UserGuide.md]: There is a typo here: it should be **Notes about the find command**.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
 
-Examples:
-* `patron add n/John s/A02128282A p/93231222 e/e03482@u.nus.edu t/student`
-* `patron add n/Alice s/S01823283S p/90123212 e/profA@u.nus.edu`
+**Notes about the add command:**
+
+* ID must be 9 characters in length. First character of ID should be 'A' and last character of ID should be an alphabet.
+* You cannot add a patron with the same NAME, EMAIL or ID as another existing patron in the patron list.
+
+**Example**
```
Comment (yl-ang) [inline docs/UserGuide.md]: I think you might want to add the following restrictions:

You can add the email length restriction.
You can also add the name length restrictions.
🤔

File: docs/UserGuide.md [hunk 4]
```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
+
+`patron delete 2`
+
+Before entering the command, the second patron in LibTask is `Bernice Yu`.
+
+![patron-delete-1](images/patron-delete-1.png)
+
+After entering the command, patron `Bernice Yu` is deleted.
+
+![patron-delete-2](images/patron-delete-2.png)
```
Comment (yuqitanyq) [inline docs/UserGuide.md:313]: Great job on specifying and elaborating on the examples here! 👍

**Review by yuqitanyq — CHANGES_REQUESTED** [review]
LGTM! Other than the small typo error, I think the PR should be good to merge! Great job! 👍

File: docs/UserGuide.md [hunk 5]
```diff
@@ -151,82 +153,185 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
 
-Examples:
-* `patron add n/John s/A02128282A p/93231222 e/e03482@u.nus.edu t/student`
-* `patron add n/Alice s/S01823283S p/90123212 e/profA@u.nus.edu`
+<div markdown="span" class="alert alert-primary">
+
+**Notes about the add command:**
+
+* ID must be 9 characters in length. First character of ID should be 'A' and last character of ID should be an alphabet.
+* You cannot add a patron with the same NAME, EMAIL or ID as another existing patron in the patron list.
```
Comment (yl-ang) [inline docs/UserGuide.md:169]: You can also add the name and email length restrictions 👍

**Review by yl-ang — APPROVED** [review]
LGTM but you can consider the suggestions in the comments 👍

**Review by yuqitanyq — APPROVED** [review]

File: docs/UserGuide.md [hunk 6]
```diff
@@ -151,82 +153,187 @@ LibTask's data is saved in the hard disk automatically after any command that ch
...
-Examples:
-* `patron add n/John s/A02128282A p/93231222 e/e03482@u.nus.edu t/student`
-* `patron add n/Alice s/S01823283S p/90123212 e/profA@u.nus.edu`
+<div markdown="span" class="alert alert-primary">
+
+**Notes about the add command:**
+
+* ID must be 9 characters in length. First character of ID should be 'A' and last character of ID should be an alphabet.
+* NAME can be a maximum of 40 characters.
+* EMAIL can be a maximum of 50 characters.
```
Comment (yl-ang) [inline docs/UserGuide.md:170]: Good job with specifying the restrictions 💯

**Review by yl-ang — APPROVED** [review]
LGTM, good job with the changes and update to the UG 👍

**Pull request merged**